tools/symbols_tool/symbols_tool.py

```diff
@@ -0,0 +1,99 @@
+# Lint as: python2, python3
+# Copyright 2021 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import argparse
+import os
+import re
+import sys
+
+from build_bazel_rules_apple.tools.wrapper_common import execute
+
+_TIMING_INFO_PATTERN = r"(.*\s\[.*,\s.*\sseconds\]:)"
+
+
+def _generate_symbols(args):
+  """Wraps bitcode_strip with the given input and output."""
+  output_dir = args.output_dir
+  archs = args.arch
+  binaries = args.binary
+
+  if not os.path.exists(output_dir):
+    os.makedirs(output_dir)
+
+  for i, arch in enumerate(archs):
+    cmd = ["xcrun",
+           "symbols",
+           "-noTextInSOD",
+           "-noDaemon",
+           "-arch",
+           arch,
+           "-symbolsPackageDir",
+           output_dir,
+           binaries[i]]
+    execute.execute_and_filter_output(
+        cmd,
+        filtering=_filter_symbols_tool_output,
```

Comment:
Is there a way to have a test for this tool, to ensure that the output is what we want?

Reply:
Added a test.